cfme/tests/automate/custom_button/test_buttons.py

```diff
@@ -77,39 +66,53 @@ def test_button_group_crud(request, appliance, obj_type):
             5. Delete the button group
             6. Assert that the button group no longer exists.
     """
-    # 1) Create it
     collection = appliance.collections.button_groups
+
+    # Create button group
     buttongroup = collection.create(
-        text=fauxfactory.gen_alphanumeric(),
-        hover=fauxfactory.gen_alphanumeric(),
+        text=fauxfactory.gen_alphanumeric(start="grp_"),
+        hover=fauxfactory.gen_alphanumeric(start="hov_"),
+        icon_color="#ff0000",
```

Comment:
Thoughts on making this the default color for the icon? So that it doesn't have to be specified each time?

Reply:
yup... agreed on default but it should be `black` `#000000` (actually default in UI side) It will provide accurate attribute :+1: